### DISCUSSION TITLE
Getting 2 notification E-Mails in one run

### DISCUSSION THREAD

**Discussion by boehser-enkel** [containrrr/watchtower#1315] [Q&A]
**Describe the bug**
--run-once with E-Mail notification: i get a Mail when the update starts (looking for container XY Updates) and another mail when the run finishes (updated XY).

**To Reproduce**
Run-once with notification

**Expected behavior**
I'd like to get one mail only for the complete run

**Screenshots**
not needed?
**Environment**
Raspbian with Docker. Everything up2date

**Comment by piksel** [comment]
You can supply [--no-startup-message](https://containrrr.dev/watchtower/arguments/#without_sending_a_startup_message) to avoid the initial message declaring what is being performed.
The message does not match the description "looking for container XY Updates", though, so if there is a different message, please quote it in the discussion.
